pkg/apply/common_test.go

```diff
@@ -64,8 +64,8 @@ func (i inventoryInfo) toUnstructured() *unstructured.Unstructured {
 	}
 }
 
-func (i inventoryInfo) toWrapped() inventory.Info {
-	return inventory.WrapInventoryInfoObj(i.toUnstructured())
+func (i inventoryInfo) toWrapped() (inventory.Info, error) {
```

Comment:
I don't really understand what this is doing.
Why does Info wrap an Unstructured?
Why does inventoryInfo contain anything other than metadata?
Why would this fail?

Reply:
I haven't fully dug into this portion of the test code, but this struct is used to set up the fake dynamic client for these tests: https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/apply/common_test.go#L292

I think this is supposed to be some glue between the dynamic client and the inventory object

Reply:
Is this going away in your rewrite?